src/components/Breadcrumbs/index.tsx

```diff
@@ -1,4 +1,4 @@
-import { ChevronRightIcon, HomeIcon } from '@heroicons/react/solid';
+import { ChevronRightIcon, HomeIcon } from '@heroicons/react/24/solid';
```

Comment:
lol, why add this "24".

Reply:
heroicons 2.0 gives you a size option but there's not much need since you can scale the components with like h-5, so I just made them all 24px. it does have more icons and redone tho so I just updated. (good to be up to date on things what can I say)